### ISSUE TITLE
double colon issue

### ISSUE THREAD

**Issue by cukal** [aefimov/idea-batch#24]
When entering :: (double colon) using the editor while editing a .cmd file I can no longer enter any characters, keyboard is unresponsive. I have to close & reopen the file. I'm using Intellij 2017.2.6 & Batch plugin 1.0.7 on MacOS.

**Comment by winthrop-polk** [comment]
I am currently experiencing the exact same issue. Did you figure out a resolution? One thing I can say for sure, everything was working perfectly... then one day I tried to enter a comment with shift+colon+colon  as usual and I think I fat figured something, either pressing a, d control, alt, or another key somewhere around that left lower part of the keyboard. That's what caused this to start happening. I started a post here:

https://intellij-support.jetbrains.com/hc/en-us/community/posts/360004274179-IntelliJ-Stopped-Editor-Working

Any help is sincerely appreciated to get this back to normal working order.

**Comment by wibotwi** [comment]
IDEA 2019.2. Entering :: does not cause an issue. Can you please check issue went away?

**Comment by winthrop-polk** [comment]
I can confirm it went away, I think I pressed a weird accidental key combination and that's what made it go away... but I'm 99% confident it could come back again if I pressed this accidental key combination again. Really wish I knew what it was... I feel as though its a conflict with intelliJ hot keys that is causing this. So to do :: you actually press and hold shift first, but with a fat finger you might have pressed other keys which, like, seems to assigned shift+:: to a hotkey that locks the page or something... it feels like something like that was happening but it's just a feeling... I can't replicate it, but this does tend to happen about once a year on average to me. If this were my app, I might consider taking a look at the similar cmd intelliJ plugin, see how they are handling comment syntax highlighting... although that plugin sucks compared to this one, I've never seen that problem there.
